ZenPacks/zenoss/Import4/rrd2tsdb.py

```diff
@@ -123,13 +135,23 @@ def _process_a_node(self, tag, tag_path, content):
             # the values need to be handled differenly
             self.type = content.strip()
             log.info("Type:%s" % self.type)
+            # if a DERIVE or COUNTER type check accompanying _GAUGE file
+            # if so, skip the traversing altogether if self.type == 'DERIVE' or self.type == 'COUNTER':
+            if self.type == 'DERIVE':
```

Comment:
Thank you for the catch. No wonder I couldn't find the changes I made. It was at the second half of the previous comment line. I'll make the change.